benchexec/tablegenerator/react-table/package.json

```diff
@@ -33,7 +48,25 @@
       "prettier/react"
     ],
     "rules": {
-      "prettier/prettier": "warn"
+      "prettier/prettier": "warn",
+      "no-console": "off",
+      "no-underscore-dangle": "off",
```

Comment:
Why are some rules turned off here? We currently have no warnings, so it should not be necessary to disable any?